.chloggen/1880.yaml

```diff
@@ -0,0 +1,8 @@
+change_type: enhancement
```

Comment:
We should mark it as breaking (breaking enhancement) since body fields will no longer be in the payload

Reply:
so what's the literal string that should go in here then?

Reply:
```suggestion
change_type: breaking 
```